client/src/redux/InteractedContacts/interactedContactsActionCreators.ts

```diff
@@ -41,7 +41,7 @@ export const getInteractedContacts = (minimalDate?: Date): ThunkAction<void, Int
 type ValueOf<T> = T[keyof T];
 
 export const setInteractedContact =
-    (id: number, propertyName: keyof GroupedInteractedContact, value: ValueOf<GroupedInteractedContact>, formState: FormState<GroupedInteractedContact>):
+    (id: number, propertyName: keyof GroupedInteractedContact, value: ValueOf<GroupedInteractedContact>, formState: FormState<GroupedInteractedContact>|null =null):
```

Comment:
why "| null = null" and not just "| null"?

Reply:
FormState<GroupedInteractedContact>|null - the type of formState
= null -the default value of formState

Reply:
👍